crates/rspack_binding_options/src/options/raw_builtins/raw_progress.rs

```diff
@@ -7,12 +7,14 @@ use serde::{Deserialize, Serialize};
 #[napi(object)]
 pub struct RawProgressPluginConfig {
   pub prefix: Option<String>,
+  pub profile: Option<bool>,
```

Comment:
Would be better to change its type to `bool`, and apply the default value `false` at js side (`boolean | undefined` -> default `false` -> `boolean` -> send to rust side by -> `bool`)

Actually `prefix` should be the same, its type should be `String`.

This is not a change request so I will still accept this PR if you don't change this, and it would be very appreciate if you could refactor this in a following PR 😃